test/Grpc.IntegrationTests/OrchestrationErrorHandling.cs

```diff
@@ -543,8 +545,8 @@ static void ValidateInnermostFailureDetailsChain(TaskFailureDetails? failureDeta
                     {
                         // Outer failure represents the orchestration failure
                         Assert.NotNull(ex.FailureDetails);
-                        Assert.True(ex.FailureDetails.IsCausedBy<TaskFailedException>());
-                        Assert.Contains("ThrowException", ex.FailureDetails.ErrorMessage);
+                        Assert.True(ex.FailureDetails.IsCausedBy<CoreSubOrchestrationFailedException>());
```

Comment:
We don't want to expose any DurableTask.Core types in the .NET Isolated model. Also, this is a breaking change so I think we need to rethink this.

Reply:
yeah what I’m mainly worried about with this change is that it looks like we don’t want to expose any core exception types here, though I’m not sure why. 
I’m totally fine with closing this. I made this change mainly because it’s what the issue/customer was asking for and with this change it seemed more in line with our in-proc model. But it seems like we just want to show that the sub-orchestrator failed in the error message, not the actual error type, is that correct?